cosmovisor/README.md

```diff
@@ -37,11 +39,9 @@ All arguments passed to the `cosmovisor` program will be passed to the current d
             └── $DAEMON_NAME
 ```
 
-Each version of the Cosmos SDK application is stored under either `genesis` or `upgrades/<name>`, which holds `bin/$DAEMON_NAME` along with any other needed files such as auxiliary client programs or libraries. `current` is a symbolic link to the currently active folder and `current/bin/$DAEMON_NAME` is the currently active binary.
-
-*Note: The `name` variable in `upgrades/<name>` holds the URI-encoded name of the upgrade as specified in the upgrade module plan.*
+The `cosmovisor/` directory incudes a subdirectory for each version of the application (i.e. `genesis` or `upgrades/<name>`). Within each subdirectory is the application binary (i.e. `bin/$DAEMON_NAME`) and any additional auxiliary files associated with each binary. `current` is a symbolic link to the currently active directory (i.e `genesis` or `upgrades/<name>`). The `name` variable in `upgrades/<name>` is the URI-encoded name of the upgrade as specified in the upgrade module plan.
```

Comment:
Yup, as mentioned above, I think it would make sense to make this change but let's double check with @robert-zaremba and then update cosmovisor to read from a `source/bin` directory rather than a `genesis/bin` directory.

Reply:
As mentioned above, we will keep "genesis binary" and instruct node operators to add an "upgrade binary" and point the current link to the "upgrade binary" instead of changing the name "genesis binary". Leaving this as is but will update the upgrade guide.